.eslintrc.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
   ],
   parser: 'babel-eslint',
   rules: {
+    'class-methods-use-this': 0,
```

Comment:
`class-methods-use-this` should be enabled. 

If a class method does not use `this`, it can safely be made a static function.